auto release doesn't work -- missing MANIFEST.in ?

this time it did produce CHANGELOG.md with version happily and a tagged release on github , but then:

```
UPDATING datalad_deprecated-0.2.1/datalad_deprecated/_version.py
set datalad_deprecated-0.2.1/datalad_deprecated/_version.py to '0.2.1'
Creating tar archive
removing 'datalad_deprecated-0.2.1' (and everything under it)
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/pep517/in_process/_in_process.py", line 363, in <module>
    main()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/pep517/in_process/_in_process.py", line 345, in main
    json_out['return_val'] = hook(**hook_input['kwargs'])
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/pep517/in_process/_in_process.py", line 130, in get_requires_for_build_wheel
    return hook(config_settings)
  File "/tmp/build-env-8nae79x2/lib/python3.7/site-packages/setuptools/build_meta.py", line 178, in get_requires_for_build_wheel
    config_settings, requirements=['wheel'])
  File "/tmp/build-env-8nae79x2/lib/python3.7/site-packages/setuptools/build_meta.py", line 159, in _get_build_requires
    self.run_setup()
  File "/tmp/build-env-8nae79x2/lib/python3.7/site-packages/setuptools/build_meta.py", line 282, in run_setup
    self).run_setup(setup_script=setup_script)
  File "/tmp/build-env-8nae79x2/lib/python3.7/site-packages/setuptools/build_meta.py", line 174, in run_setup
    exec(compile(code, __file__, 'exec'), locals())
  File "setup.py", line 7, in <module>
    from _datalad_buildsupport.setup import (
ModuleNotFoundError: No module named '_datalad_buildsupport'
* Creating venv isolated environment...
* Installing packages in isolated environment... (setuptools >= 30.3.0, wheel)
* Getting dependencies for sdist...
* Installing packages in isolated environment... (setuptools >= 30.3.0)
* Building sdist...
* Building wheel from sdist
* Creating venv isolated environment...
* Installing packages in isolated environment... (setuptools >= 30.3.0, wheel)
* Getting dependencies for wheel...

ERROR Backend subproccess exited when trying to invoke get_requires_for_build_wheel
✖  error     Error: Command failed: python -m build && twine upload dist/* 
    at checkExecSyncError (child_process.js:630:11)
    at Object.execSync (child_process.js:666:15)
    at Object.execSync (pkg/prelude/bootstrap.js:1623:30)
    at runExecSync (/snapshot/auto/plugins/exec/dist/index.js:54:43)
    at /snapshot/auto/plugins/exec/dist/index.js:129:52
    at eval (eval at create (/snapshot/auto/node_modules/tapable/lib/HookCodeFactory.js:75:10), <anonymous>:21:1)
    at new Promise (<anonymous>)
    at Hook.eval [as promise] (eval at create (/snapshot/auto/node_modules/tapable/lib/HookCodeFactory.js:75:10), <anonymous>:4:8)
    at Hook.PROMISE_DELEGATE [as _promise] (/snapshot/auto/node_modules/tapable/lib/Hook.js:22:14)
    at Auto.makeRelease (/snapshot/auto/node_modules/@auto-it/core/src/auto.ts:2024:37)
Error: Process completed with exit code 1.

```

<details>
<summary>python3 setup.py sdist worked fine for me locally</summary> 

```shell
(git)lena:~datalad/datalad-deprecated-clone[tags/0.2.1^0]
$> python3 setup.py sdist
running sdist
running egg_info
creating datalad_deprecated.egg-info
writing datalad_deprecated.egg-info/PKG-INFO
writing dependency_links to datalad_deprecated.egg-info/dependency_links.txt
writing entry points to datalad_deprecated.egg-info/entry_points.txt
writing requirements to datalad_deprecated.egg-info/requires.txt
writing top-level names to datalad_deprecated.egg-info/top_level.txt
writing manifest file 'datalad_deprecated.egg-info/SOURCES.txt'
reading manifest file 'datalad_deprecated.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
adding license file 'LICENSE'
adding license file 'COPYING'
writing manifest file 'datalad_deprecated.egg-info/SOURCES.txt'
running check
creating datalad_deprecated-0.2.1
creating datalad_deprecated-0.2.1/datalad_deprecated
creating datalad_deprecated-0.2.1/datalad_deprecated.egg-info
creating datalad_deprecated-0.2.1/datalad_deprecated/resources
creating datalad_deprecated-0.2.1/datalad_deprecated/resources/website
creating datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets
creating datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/css
creating datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/images
creating datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/js
creating datalad_deprecated-0.2.1/datalad_deprecated/resources/website/tests
creating datalad_deprecated-0.2.1/datalad_deprecated/resources/website/tests/qunit
creating datalad_deprecated-0.2.1/datalad_deprecated/tests
copying files to datalad_deprecated-0.2.1...
copying COPYING -> datalad_deprecated-0.2.1
copying LICENSE -> datalad_deprecated-0.2.1
copying MANIFEST.in -> datalad_deprecated-0.2.1
copying README.md -> datalad_deprecated-0.2.1
copying pyproject.toml -> datalad_deprecated-0.2.1
copying setup.cfg -> datalad_deprecated-0.2.1
copying setup.py -> datalad_deprecated-0.2.1
copying versioneer.py -> datalad_deprecated-0.2.1
copying datalad_deprecated/__init__.py -> datalad_deprecated-0.2.1/datalad_deprecated
copying datalad_deprecated/_version.py -> datalad_deprecated-0.2.1/datalad_deprecated
copying datalad_deprecated/annotate_paths.py -> datalad_deprecated-0.2.1/datalad_deprecated
copying datalad_deprecated/auto.py -> datalad_deprecated-0.2.1/datalad_deprecated
copying datalad_deprecated/diff.py -> datalad_deprecated-0.2.1/datalad_deprecated
copying datalad_deprecated/gitrepo.py -> datalad_deprecated-0.2.1/datalad_deprecated
copying datalad_deprecated/ls.py -> datalad_deprecated-0.2.1/datalad_deprecated
copying datalad_deprecated/ls_webui.py -> datalad_deprecated-0.2.1/datalad_deprecated
copying datalad_deprecated/publish.py -> datalad_deprecated-0.2.1/datalad_deprecated
copying datalad_deprecated/sibling_webui.py -> datalad_deprecated-0.2.1/datalad_deprecated
copying datalad_deprecated/utils.py -> datalad_deprecated-0.2.1/datalad_deprecated
copying datalad_deprecated.egg-info/PKG-INFO -> datalad_deprecated-0.2.1/datalad_deprecated.egg-info
copying datalad_deprecated.egg-info/SOURCES.txt -> datalad_deprecated-0.2.1/datalad_deprecated.egg-info
copying datalad_deprecated.egg-info/dependency_links.txt -> datalad_deprecated-0.2.1/datalad_deprecated.egg-info
copying datalad_deprecated.egg-info/entry_points.txt -> datalad_deprecated-0.2.1/datalad_deprecated.egg-info
copying datalad_deprecated.egg-info/requires.txt -> datalad_deprecated-0.2.1/datalad_deprecated.egg-info
copying datalad_deprecated.egg-info/top_level.txt -> datalad_deprecated-0.2.1/datalad_deprecated.egg-info
copying datalad_deprecated/resources/website/index.html -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website
copying datalad_deprecated/resources/website/assets/css/jquery.dataTables-1.10.12.css -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/css
copying datalad_deprecated/resources/website/assets/css/main.css -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/css
copying datalad_deprecated/resources/website/assets/images/external-link-symbol.svg -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/images
copying datalad_deprecated/resources/website/assets/images/favicon-16x16.png -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/images
copying datalad_deprecated/resources/website/assets/images/favicon.ico -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/images
copying datalad_deprecated/resources/website/assets/images/favicon.svg -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/images
copying datalad_deprecated/resources/website/assets/images/logo_wide.svg -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/images
copying datalad_deprecated/resources/website/assets/images/sort_asc.png -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/images
copying datalad_deprecated/resources/website/assets/images/sort_both.png -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/images
copying datalad_deprecated/resources/website/assets/images/sort_desc.png -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/images
copying datalad_deprecated/resources/website/assets/js/jquery-2.2.4.js -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/js
copying datalad_deprecated/resources/website/assets/js/jquery.dataTables-1.10.12.js -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/js
copying datalad_deprecated/resources/website/assets/js/main.js -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/js
copying datalad_deprecated/resources/website/assets/js/md5-2.3.0.js -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/js
copying datalad_deprecated/resources/website/assets/js/showdown-1.8.5-33-g0c6ce94.js -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/assets/js
copying datalad_deprecated/resources/website/tests/test.html -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/tests
copying datalad_deprecated/resources/website/tests/qunit/qunit-2.0.1.css -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/tests/qunit
copying datalad_deprecated/resources/website/tests/qunit/qunit-2.0.1.js -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/tests/qunit
copying datalad_deprecated/resources/website/tests/qunit/sinon-1.17.5.js -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/tests/qunit
copying datalad_deprecated/resources/website/tests/qunit/sinon-qunit-1.0.0.js -> datalad_deprecated-0.2.1/datalad_deprecated/resources/website/tests/qunit
copying datalad_deprecated/tests/__init__.py -> datalad_deprecated-0.2.1/datalad_deprecated/tests
copying datalad_deprecated/tests/test__main__.py -> datalad_deprecated-0.2.1/datalad_deprecated/tests
copying datalad_deprecated/tests/test_annexrepo.py -> datalad_deprecated-0.2.1/datalad_deprecated/tests
copying datalad_deprecated/tests/test_annotate_paths.py -> datalad_deprecated-0.2.1/datalad_deprecated/tests
copying datalad_deprecated/tests/test_auto.py -> datalad_deprecated-0.2.1/datalad_deprecated/tests
copying datalad_deprecated/tests/test_create_sibling_webui.py -> datalad_deprecated-0.2.1/datalad_deprecated/tests
copying datalad_deprecated/tests/test_diff.py -> datalad_deprecated-0.2.1/datalad_deprecated/tests
copying datalad_deprecated/tests/test_gitrepo.py -> datalad_deprecated-0.2.1/datalad_deprecated/tests
copying datalad_deprecated/tests/test_ls.py -> datalad_deprecated-0.2.1/datalad_deprecated/tests
copying datalad_deprecated/tests/test_ls_webui.py -> datalad_deprecated-0.2.1/datalad_deprecated/tests
copying datalad_deprecated/tests/test_publish.py -> datalad_deprecated-0.2.1/datalad_deprecated/tests
copying datalad_deprecated/tests/test_utils.py -> datalad_deprecated-0.2.1/datalad_deprecated/tests
Writing datalad_deprecated-0.2.1/setup.cfg
UPDATING datalad_deprecated-0.2.1/datalad_deprecated/_version.py
set datalad_deprecated-0.2.1/datalad_deprecated/_version.py to '0.2.1'
creating dist
Creating tar archive
removing 'datalad_deprecated-0.2.1' (and everything under it)

```
</details>

I see that datalad-container (where auto was established before) has MANIFEST.in , and it was also added by @jwodder to MANIFEST.in in the -template (we should look into cookie cutting the template to allow for "upgrades").  Will add it and hope that it would fix this issue

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,4 +1,6 @@
-include versioneer.py
-include datalad_deprecated/_version.py
-
+include CONTRIBUTORS LICENSE versioneer.py
+graft _datalad_buildsupport
 graft datalad_deprecated/resources
+graft docs
+prune docs/build
+global-exclude *.py[cod]

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,2 +1,2 @@
 [build-system]
-requires = ["setuptools >= 30.3.0", "wheel"]
+requires = ["setuptools >= 43.0.0", "wheel"]

--- a/setup.py
+++ b/setup.py
@@ -12,9 +12,10 @@ cmdclass = versioneer.get_cmdclass()
 cmdclass.update(build_manpage=BuildManPage)
 
 # Give setuptools a hint to complain if it's too old a version
-# 30.3.0 allows us to put most metadata in setup.cfg
+# 43.0.0 allows us to put most metadata in setup.cfg and causes pyproject.toml
+# to be automatically included in sdists
 # Should match pyproject.toml
-SETUP_REQUIRES = ['setuptools >= 30.3.0']
+SETUP_REQUIRES = ['setuptools >= 43.0.0']
 # This enables setuptools to install wheel on-the-fly
 SETUP_REQUIRES += ['wheel'] if 'bdist_wheel' in sys.argv else []
 

